meshery.sh

```diff
@@ -35,7 +35,12 @@ main() {
 	kubectl config view --minify --flatten > ~/minified_config
 	mv ~/minified_config ~/.kube/config
 
-  curl -L https://git.io/meshery | PLATFORM=$PLATFORM bash -
+	curl -L https://git.io/meshery | DEPLOY_MESHERY=false bash -
+
+	mesheryctl system context create new-context --platform $PLATFORM --url http://localhost:9081 --set --yes
+	mesheryctl system channel set edge-latest
```

Comment:
Could we replace this with `mesheryctl system start -p $PLATFORM`?

Why do we need to use the edge release? We should use stable IMO.

Reply:
Makes sense :heavy_check_mark: